lib/helpers.js

```diff
@@ -11,8 +12,7 @@ export function notifyAllowWallet({account, preventRenotify = false}) {
   if(preventRenotify && notifiedAccounts.get(account)) {
     return;
   }
-  // if inside a WebView ignore
-  if(isWebView({userAgent: navigator?.userAgent})) {
+  if(config?.vueWallet?.disableChapi === true) {
```

Comment:
Does this need to explicitly check for `true` or is truthy good enough?
```suggestion
  if(config?.vueWallet?.disableChapi) {
```